Formula/s/sshx.rb

```diff
@@ -0,0 +1,21 @@
+class Sshx < Formula
+  desc "Fast, collaborative live terminal sharing over the web"
+  homepage "https://sshx.io"
+  url "https://github.com/ekzhang/sshx/archive/refs/tags/v0.3.1.tar.gz"
+  sha256 "3bc2abc59be26c9eefaf7fc999aa4a3723d2916414c319fd06555415278acb64"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "protobuf"
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "crates/sshx"
+  end
+
+  test do
+    process = IO.popen("#{bin}/sshx --quiet")
+    sleep 1
+    Process.kill "TERM", process.pid
+    assert_match "https://sshx.io/s/", process.read
+  end
```

Comment:
```suggestion
  test do
    process = IO.popen("#{bin}/sshx --quiet")
    sleep 1
    assert_match "https://sshx.io/s/", process.read
  ensure
    Process.kill("TERM", process.pid)
    Process.wait(process.pid)
  end
```

Reply:
This never terminates, but it works when I move the `Process.kill` outside of the `ensure` block.